lib/internal/fs/watchers.js

```diff
@@ -11,41 +11,44 @@ const {
   getStatsFromBinding,
   validatePath
 } = require('internal/fs/utils');
+const {
+  defaultTriggerAsyncIdScope
+} = require('internal/async_hooks');
 const { toNamespacedPath } = require('path');
 const { validateUint32 } = require('internal/validators');
 const { getPathFromURL } = require('internal/url');
 const util = require('util');
 const assert = require('assert');
 
+const kOldStatus = Symbol('kOldStatus');
+const kUseBigint = Symbol('kUseBigint');
+const kOwner = Symbol('kOwner');
```

Comment:
Not a big deal, but don't we usually use a normal `owner` property on the handles elsewhere?

Reply:
We use a symbol in (most/all) new code.

Reply:
Yes, I’d like to transition to using a single `kOwner` symbol for all native handles

Reply:
That's a good idea (and probably what would have been done if symbols existed back in the day).